lib/instrumentation/generic-span.js

```diff
@@ -101,3 +109,9 @@ GenericSpan.prototype.addLabels = function (labels) {
   }
   return true
 }
+
+GenericSpan.prototype.setType = function (type, subtype, action) {
```

Comment:
To make sure the default values are always `null`, I think we can do this:

```suggestion
GenericSpan.prototype.setType = function (type, subtype = null, action = null) {
```

Reply:
fixed